service/templates/task.html

```diff
@@ -242,6 +242,10 @@ <h1 class="one-liner" id="task-title">
                       <td class="icon"><i class="fa fa-calendar"></i></td>
                       <td>Finished on [[task.finish_time | dateFromNanoseconds | convertDateToUserTimezone:userTz:"MMM D, YYYY h:mm:ss a"]]</td>
                     </tr>
+                    <tr ng-show="task.task_end_details.status == 'failed' && (task.task_end_details.type == 'setup' || task.task_end_details.type == 'system') ">
+                      <td class="icon"><i class="fa fa-file-text-o"</td>
```

Comment:
Maybe use "code" instead of "file-text-o"? It would look like this

![image](https://user-images.githubusercontent.com/29384396/52740979-52b03d80-2fa2-11e9-9f0d-38e9a135c83a.png)

Reply:
almost as good as spock hand 👍